runelite-client/src/main/java/net/runelite/client/plugins/slayer/TargetClickboxOverlay.java

```diff
@@ -75,11 +88,28 @@ private void renderTargetOverlay(Graphics2D graphics, NPC actor, Color color)
 		Shape objectClickbox = actor.getConvexHull();
 		if (objectClickbox != null)
 		{
-			graphics.setColor(color);
-			graphics.setStroke(new BasicStroke(2));
-			graphics.draw(objectClickbox);
-			graphics.setColor(ColorUtil.colorWithAlpha(color, color.getAlpha() / 12));
-			graphics.fill(objectClickbox);
+			renderPoly(graphics, objectClickbox, color);
 		}
 	}
+
+	private void renderTargetTiles(Graphics2D graphics, NPC actor, Color color)
+	{
+		NPCComposition npcComposition = actor.getTransformedComposition();
+		if (npcComposition != null)
+		{
+			int size = npcComposition.getSize();
+			LocalPoint lp = actor.getLocalLocation();
+			Polygon tilePoly = Perspective.getCanvasTileAreaPoly(client, lp, size);
+			renderPoly(graphics, tilePoly, color);
+		}
+	}
+
+	private void renderPoly(Graphics2D graphics, Shape shape, Color color)
+	{
+		graphics.setColor(color);
+		graphics.setStroke(new BasicStroke(2));
+		graphics.draw(shape);
+		graphics.setColor(new Color(color.getRed(), color.getGreen(), color.getBlue(), 20));
```

Comment:
... what is this?

Reply:
This was taken from `renderPoly` in `NpcSceneOverlay.java`

I'll adjust this to match what was here before instead